js/app.js

```diff
@@ -0,0 +1,483 @@
+// console.log('Homework!!')
+
+// Q & A
+
+// 1. We assign a variable by declaring it within the js document.
+// 	  You can declare a variable with var, let, or const. Each one 
+// 	  has its uses except var... never use var... you can also declare
+//	  the variable without any of these, but that's like declaring it
+//	  var. Don't do it.
+
+// 2. If you declare your variable with let, you can redeclare it later
+//	  in your code to have another value by simply rewriting it or by
+//	  manipulating the data within it using various means. If
+//	  you declare it with const, you can only update the value by
+//	  manipulating the data inside of it. It cannot be redeclared
```

Comment:
you **cannot** redeclare with `let`, only *reassign*.  
```
let x = 5;
let x = 6;
```
will cause an error.  
however, you _can_ reassign, like so: 
```
let x = 5;
x = 6
```